core/src/main/java/org/geysermc/geyser/util/DimensionUtils.java

```diff
@@ -41,7 +41,7 @@
 public class DimensionUtils {
 
     // Changes if the above-bedrock Nether building workaround is applied
-    private static int BEDROCK_NETHER_ID = 1;
+    public static int BEDROCK_NETHER_ID = 1;
```

Comment:
Can you change this to private and add a static getter? PR looks good otherwise.

Reply:
I added a boolean method to cut down on magic numbers in random parts of the codebase. I didn't mean to publish the `Set` to `List` change but it doesn't matter as iirc ArrayList is ordered and I'm pretty sure that's how fog ids should be stored.

Reply:
I would revert that change because `Set`s cannot have duplicate values in it which corresponds with my understanding of the fog stack.

Reply:
Changed it back, we can look into it more later. It works so it's fine.